pkgs/by-name/ac/actual-server/package.nix

```diff
@@ -76,14 +95,57 @@ let
 
     outputHashAlgo = "sha256";
     outputHashMode = "recursive";
-    outputHash =
-      {
-        aarch64-darwin = "sha256-IJBfBA71PZeE/Zlu2kzQw8l/D4lVAV5I5loRyRfncKA=";
-        aarch64-linux = "sha256-djE2lt/o/7kd7ci2TW3mhjSptD3etChbvtdbiWqp/wo=";
-        x86_64-darwin = "sha256-AShd87VFwqDbJZoFJPg6HsdhTx7XMVdZ5sRWLXU8ldM=";
-        x86_64-linux = "sha256-me0v+RuoleOKFRyJ7iyLTKRnV2Cz2Q1MLc/SE2sSSH8=";
-      }
-      .${stdenv.hostPlatform.system} or (throw "Unsupported system: ${stdenv.hostPlatform.system}");
+    outputHash = "sha256-Tac2gOkdc2tzNKB3ARMfJad1MkOphudvN74gI8bGMtY=";
+  };
+
+  webUi = stdenvNoCC.mkDerivation {
+    name = "actual-server-${version}-webui";
```

Comment:
this should probably be split into pname and version (see https://github.com/NixOS/nixpkgs/issues/103997)

Reply:
I don't know... I followed `offlineCache` for this because this derivation also won't be on end user's system (it's copied in `installPhase`). Also this name doesn't follow usual `${pname}-${version}` pattern.

Reply:
What do you think with my explanation?

Reply:
Not sure - why is the different naming scheme needed?

I generally thought that packages should use pname + version instead of name whenever possible. If there's a reason for doing it differently I'd love to know, but as far as I can tell right now there doesn't seem to be any benefit to using `name` in this case. I'd personally stick to pname and version if possible since that is the standard practice, but I'm open to doing it differently if there's a technical reason.

Reply:
Well... Ok then. Changed to `pname` and `version`.